src/shared/ui/LoginRequiredUi/RequireLoginDialog.tsx

```
@@ -1,19 +1,51 @@
+                </Suspense>
+              </div>
+            </div>
+          </div>
+          {/* X 버튼 */}
+          <button
+            onClick={onClose}
+            className="absolute -top-3 -right-3 w-8 h-8 bg-[#214d35] hover:bg-[#1a2332] rounded-full flex items-center justify-center text-white text-lg font-bold transition-colors"
+            aria-label="닫기"
+          >
+            ×
+          </button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)